app/views/posts/show.html.slim

```diff
@@ -1,17 +1,41 @@
+= stylesheet_link_tag 'comments'
+
 .dflex.justify-content-center
   .card style="padding:2% 5% 2% 5%"
-    small.text-muted
-      | Posted by&#160
-      i
+    span
```

Comment:
span ir inline tipa wrapperis, īsti neredzu kādēļ tev šeit šis bija nepieciešams :)

Reply:
Ideja bija , ka edit un delete pogas ir blakus vardam, bet beigas neizskatijas tik labi ka iedomajos, tad beigas salaboju lai izskatas kaut cik piedienigi un par so span aizmirsu.